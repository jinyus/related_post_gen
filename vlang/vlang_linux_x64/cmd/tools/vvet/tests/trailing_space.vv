// NB: This file has and *should* have trailing spaces.
// When making changes, please ensure they are not removed.

fn after_comments() {
	// spaces after line comments give errors  
	/*  
	in block comments  
	too  
	*/  
}

fn main() {
	var := 'error about the spaces right there'  
	no_err := "inside multi line strings it's fine.  
but not after"  
}
