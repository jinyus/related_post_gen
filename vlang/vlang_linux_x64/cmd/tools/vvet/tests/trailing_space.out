cmd/tools/vvet/tests/trailing_space.vv:5: error: Looks like you have trailing whitespace.
cmd/tools/vvet/tests/trailing_space.vv:6: error: Looks like you have trailing whitespace.
cmd/tools/vvet/tests/trailing_space.vv:7: error: Looks like you have trailing whitespace.
cmd/tools/vvet/tests/trailing_space.vv:8: error: Looks like you have trailing whitespace.
cmd/tools/vvet/tests/trailing_space.vv:9: error: Looks like you have trailing whitespace.
cmd/tools/vvet/tests/trailing_space.vv:13: error: Looks like you have trailing whitespace.
cmd/tools/vvet/tests/trailing_space.vv:15: error: Looks like you have trailing whitespace.
